.github/workflows/main.yml

```diff
@@ -20,7 +21,7 @@ jobs:
         id: ipfs
         uses: aquiladev/ipfs-action@master
         with:
-          path: ../mips
```

Comment:
@sanbotto I think to trigger also when this workflow file changes, because there might be changes in this file that you would want to deploy, e.g. name changes, var changes, etc.

Reply:
@cristidas this is not a trigger, it's just the path to the dir that should be uploaded. `../mips` means the whole repo, and that's why I'm doing a sparse checkout in line 18, to only upload what makes sense to upload. The workflow gets triggered for every push to `master`.